test/testpartner.ts

```diff
@@ -109,6 +155,8 @@ async function main(): Promise<void> {
   const avaxWallet = await account.waitForCurrencyWallet(avaxInfo?.id ?? '')
   const arrrWallet = await account.waitForCurrencyWallet(arrrInfo?.id ?? '')
   const maticWallet = await account.waitForCurrencyWallet(maticInfo?.id ?? '')
+  await ethWallet.changePaused(false)
+  await avaxWallet.changePaused(false)
```

Comment:
Why are these wallets un-paused? Comment?

Reply:
Wallets are now paused by default so you have to unpause them to use them.

Reply:
Should this be an argument to the CLI?